src/main/java/com/rtb/projectmanagementtool/user/UserController.java

```diff
@@ -13,6 +13,20 @@ public UserController(DatastoreService datastore) {
     this.datastore = datastore;
   }
 
+  // Only way to find users to add to projects currently
+  // is with their name; means that we'd need to make sure there
```

Comment:
This is a dangerous assumption.  In a school setting, chances that at least 2 people have the same name are really high.  Probably would be good to come up with a way to get this to work.